Sources/TuistLoader/Loaders/ManifestLoader.swift

```diff
@@ -163,6 +163,10 @@ public class ManifestLoader: ManifestLoading {
             let manifestPath = path.appending(component: fileName)
             if !FileHandler.shared.exists(manifestPath) { continue }
             let data = try loadManifestData(at: manifestPath)
+            if Environment.shared.isVerbose {
+                let string = String(data: data, encoding: .utf8)
+                logger.debug("Could not load manifest, represented by this JSON:\n\(string)")
```

Comment:
At this point it's not true that we could not load the manifest. The line to load the manifest, and that could potentially raise an error is right below this. I'd change the message to:

```
"Trying to load parse the manifest represented by the following JSON representation:\n\(string)"
```